libraries/AP_Scripting/generator/src/main.c

```diff
@@ -1956,6 +1956,14 @@ void emit_argcheck_helper(void) {
   fprintf(source, "}\n\n");
 }
 
+void emit_not_supported_helper(void) {
+  fprintf(source, "static int not_supported_error(lua_State *L, int expected_arg_count, const char* name) {\n");
```

Comment:
expected arg count is wrong, it's actually the argument index on the stack, IE it should just be be arg here, not `expected_arg_count`, that implies something different then what it actually is.

Reply:
fixed